src/pages/Profile.tsx

```diff
@@ -653,6 +681,49 @@ class Profile extends Component<Props, ProfileState> {
           )}
         </div>
 
+        <div className="row">
+          <div className="col-md-12 text-center">
+            <button
+              onClick={() => {
+                this.loadUserSubscriptions(
+                  this.state.subscriptionsOffset + subscriptionsLimit
+                );
+              }}
+              hidden={
+                !this.state.hasMoreSubscriptions ||
+                (this.state.areSubscriptionsLoading &&
+                  this.state.hasMoreSubscriptions)
```

Comment:
You don't need to check `this.state.hasMoreSubscriptions` twice
```suggestion
                !this.state.hasMoreSubscriptions || this.state.areSubscriptionsLoading
```